src/posix/platform/openthread-posix-config.h

```diff
@@ -229,4 +229,15 @@
 #error "OPENTHREAD_CONFIG_POSIX_TREL_USE_NETLINK_SOCKET was removed (no longer applicable with TREL over DNS-SD)."
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT
+ *
+ * This setting configures the TREL UDP port number.
+ * Define as 0 to use an ephemeral port number.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT
+#define OPENTHREAD_CONFIG_POSIX_TREL_UDP_PORT 0
+#endif
```

Comment:
> @simonlingoogle, code-wise it looks good, but I am wondering what is the background for adding  this?
> 
> 

using dynamic port would cause it difficult to add iptable rules to unblock the udp port.

Reply:
It should be case by case.